test/graphql/resolvers/event_resolver_test.rb

```diff
@@ -10,19 +10,19 @@
   describe '.create' do
     it 'creates a new event' do
       timestamp = Time.now.utc
-      attrs = {
-        'title' => 'new',
-        'description' => 'newd',
-        'eventType' => { 'id' => type.id },
-        'organization' => { 'id' => org.id },
-        'office' => { 'id' => office.id },
-        'location' => 'newl',
-        'startsAt' => timestamp.iso8601,
-        'endsAt' => (timestamp + 10.minutes).iso8601,
-        'capacity' => 10,
-      }
-
-      EventResolver.create(nil, {input: attrs}, nil)
+      input = stub(
```

Comment:
why are we stubbing now?

Reply:
forgive my forgetfulness about `stub`. Is it any different from using `OpenStruct.new()`? Any performance penalties? If we're not using the `stub` API methods, maybe tests are faster with a simple struct?

Reply:
`stub` is a common testing pattern. in other frameworks it may be called `mock` or `double`. I don't want to switch to OpenStruct because it's not very common. Also OpenStruct is different that Struct and I don't think it would be any faster than a stub.